internal/pkg/dl/constants.go

```diff
@@ -50,7 +50,6 @@ const (
 	FieldUnenrolledAt     = "unenrolled_at"
 	FieldUpgradedAt       = "upgraded_at"
 	FieldUpgradeStartedAt = "upgrade_started_at"
-	FieldUpgradeStatus    = "upgrade_status"
```

Comment:
@michel-laterman should I leave in this field? As you will use it in https://github.com/elastic/fleet-server/pull/1887